src/utils/SqsClient.ts

```diff
@@ -1,18 +1,29 @@
 import { SQSClient, SendMessageCommand } from "@aws-sdk/client-sqs";
 import { NodeHttpHandler } from "@aws-sdk/node-http-handler";
 import AWSXRay from "aws-xray-sdk-core";
+import { mockSqsCient } from "../tests/contract/mocks/sqsClient";
 
-AWSXRay.setContextMissingStrategy("LOG_ERROR");
+const createSqsClient = () => {
+	let sqsClient: SQSClient;
+	if (process.env.USE_MOCKED) {
+		console.log("USING MOCKED");
```

Comment:
Do console.logs come through to splunk? Just thinking that obviously weren't not expecting this case to execute in production but if something went really wrong we'd want to know about that

Reply:
I am not 100% sure on this, may depend on the query we use to retrieve these logs? @ViveakV I left these logs thinking it might be useful while we run the tests locally.

Reply:
Added logger instead of console log as discussed.